softwarereview_policies.Rmd

```diff
@@ -60,14 +60,17 @@ Note that not all rOpenSci projects and packages are in-scope or go through peer
 
 ### Package categories {#package-categories}
 
-- **data retrieval**: Packages for accessing and downloading data from online sources with scientific applications.  Our definition of scientific applications is broad, including data storage services, journals, and other remote servers, as many data sources may be of interest to researchers. However, retrieval packages should be focused on data *sources* / *topics*, rather than *services*. For example a general client for Amazon Web Services data storage would not be in-scope. (Examples: [**rotl**](https://github.com/ropensci/software-review/issues/17),
-  [**gutenbergr**](https://github.com/ropensci/software-review/issues/41))
+- **data retrieval**: Packages for accessing and downloading data from online sources with scientific applications.
+  Our definition of scientific applications is broad, including data storage services, journals, and other remote servers, as many data sources may be of interest to researchers.
+  However, retrieval packages should be focused on data *sources* or *topics*, rather than *services*, and should do [more than just download data](https://ropensci.org/blog/2022/06/16/publicize-api-client-yes-no).
+  For example a general client for Amazon Web Services data storage would not be in-scope, nor would a package which only offered download functionality without any pre-processing or pre-filtering.
```

Comment:
Would https://docs.ropensci.org/riem/ still be in scope? Really minimal processing, just checking an airport exists for instance.